private/scripts/helpers.sh

```diff
@@ -281,10 +281,14 @@ function update_php() {
 # Install sage and related dependencies.
 function install_sage_theme() {
   # Check if the directory $sagedir is empty. If it's not, bail.
-  echo "Checking if ${sagedir} is exists and if it's empty."
+  echo "Checking if ${sagedir} exists and if it's empty."
 
   if [ "$(ls -A "$sagedir")" ]; then
     echo "${red}Directory not empty!${normal}"
+    if [ "$is_ci" -eq 1 ]; then
```

Comment:
```suggestion
    if [[ "$is_ci" == 1 ]]; then
```
-eq will error if is_ci is an empty string. I don't see where it's coming from though so possibly this is unnecessary.